src/sql/command.py

```diff
@@ -89,3 +87,55 @@ def connection(self):
     def result_var(self):
         """Returns the result_var"""
         return self.parsed["result_var"]
+
+    def _params_helper(self, params):
+        def is_float(element):
+            if element is None:
+                return False
+            try:
+                float(element)
+                return True
+            except ValueError:
+                return False
+
+        params_dict = {}
+        if not params:
+            return params_dict
+            # Throw some exception
+
+        for key, value in params:
+            if is_float(value):
+                params_dict[key] = int(value)
+            else:
+                params_dict[key] = value
+        return params_dict
+
+    def _var_expand(self, magic, user_ns, line, cell):
+        """
+        Support for the variable substition in the SQL clause
+        For now, we have enabled two ways:
+        1. Latest format, {{a}}, we use jinja2 to parse the string with {{a}} format
+        2. Legacy format, {a}, $a, and :a format.
+
+        We will deprecate the legacy format feature in next major version
+        """
+        self.is_legacy_var_expand_parsed = False
+        # Latest format parsing
+        # TODO: support --param and --use-global logic here
```

Comment:
add a link to the issue so we know what this is referring to

Reply:
Fix: [269447d](https://github.com/ploomber/jupysql/pull/137/commits/269447d07a10fa5579906fd345f70d45b655f653)